api/cloudConfigRule.go

```diff
@@ -83,7 +83,7 @@ query GetCloudConfigRule($id: ID!) {
 type CloudConfigRule struct {
 	BuiltIn                 bool                               `json:"builtin"`
 	CloudProvider           string                             `json:"cloudProvider"`
-	Control                 []CloudConfigRuleQueryControl      `json:"control"`
+	Control                 *CloudConfigRuleQueryControl       `json:"control"`
```

Comment:
```suggestion
	Control                 CloudConfigRuleQueryControl       `json:"control"`
```

Reply:
We can have the `null` value for the property, if it is not available for the resource/resolver.